mantidproject/mantid
# Magnetic Order Parameter Fit Function

**Description of work.**
Added a new Magnetic Order Parameter Fit function for Muon modelling and associated documentation.

**To test:**
1. Ensure you have Feature Flags for Model Fitting enabled (see #32572 for more details)
2. Open ```Muon Analysis GUI```
2. Load EMU 96133
3. Click on the ```Grouping Tab``` and click ```Guess``` in the Guess Alpha column
4. Load runs EMU 96116-96132
5. Go to the ```Fitting Tab```
6. Add a ```GausOsc``` Fit and set the parameters as follows: ```A = 0.03, Sigma = 0.5, Frequency = 2.5, Phi = 0```
7. Add a ```StretchExpMuon``` fit
8. Click on the ```Sequential Fitting Tab``` and click ```Sequentially Fit all``` button
9. Click on the ```Results Tab``` and select sample_temp from the log list. Click ```Output Results```
10. Click on the Model Fitting Tab. Set x to be sample_temp and y to be cost function
11. Add a Magentic Order Parameter Fit function and fit to the data

Fixes #32583 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
